.github/workflows/spark-integration-test.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       matrix:
         architecture: [linux/amd64]
         java-version: [ 8, 11, 17 ]
-        scala-version: [ 2.12 ]
+        scala-version: [ 2.12, 2.13 ]
```

Comment:
@FANNG1 should we add the scala2.13 jobs to the pipeline?

Reply:
There are too many pipelines for now, let's check the test result of the ci

Reply:
There are test errors in the ci, could you fix it?

Reply:
> There are test errors in the ci, could you fix it?

sure